StripeFinancialConnections/StripeFinancialConnections/Source/Native/Shared/CreatePaneParameters.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  CreatePaneParameters.swift
+//  StripeFinancialConnections
+//
+//  Created by Krisjanis Gaidis on 7/22/24.
+//
+
+import Foundation
+
+// A bag of extra parameters we can pass to `CreatePaneViewController` function
+// that creates new pane view controllers. This avoids preserving state in
+// `NativeFlowDataManager` where the state might be outdated after a specific
+// pane push.
+struct CreatePaneParameters {
```

Comment:
historically, lots of parameters were set on `NativeFlowDataManager` (you can see a bunch of them now)

however, at least for some parameters, this was likely not the best choice as its potentially preserving state that is no longer true after some pane pushes

that being said, this was never such a clear problem until this feature where preserving `nextPaneOrDrawerOnSecondaryCta` can actually cause a bug:
1. Tap "Manually verify instead" in Consent
2. Drawer presents and we save `nextPaneOrDrawerOnSecondaryCta`
3. Exit the drawer
4. Press "Agree and conitnue"
5. See drawer
6. Press "Not Now"
7. Pressing "Not Now" will wrongly navigate to manual entry instead of institution picker (due to actions in (2))

Reply:
[PR with similar change in Stripe.js](https://git.corp.stripe.com/stripe-internal/stripe-js-v3/pull/23245/files)